### DISCUSSION TITLE
reading 3D properties

### DISCUSSION THREAD

**Discussion by MarkusZechner** [equinor/resdata#904] [General]
Hi all!

Thanks for this repo!

Does anyone have a tutorial on reading static and dynamic 3D reservoir properties (e.g., porosity and saturations)?

Thank you!!

Best,

Markus

**Comment by berland** [comment]
I would recommend you to try out `ecl2df`, and its associated documentation at https://equinor.github.io/ecl2df/

**Reply by MarkusZechner** [reply]
Thank you very much for your response! Is there any workaround to use it for Intersect as well? I think the binary files are the same, but the library requires to define the data deck (`eclfiles = ecl2df.EclFiles("MYECLDECK.DATA")`), which isn't available in intersect.
Thank you again!

Best,

Markus
